Fix PHP warning

fernandomema opened the pull request:
Fix PHP warning: 
Warning: Invalid argument supplied for foreach() in %htdocs_path%\vendor\picorm\picorm\src\PicORM\Model.php on line 921

fernandomema committed:
Fix PHP warning

Fix PHP warning: 
Warning: Invalid argument supplied for foreach() in %htdocs_path%\vendor\picorm\picorm\src\PicORM\Model.php on line 921

iNem0o commented on src/PicORM/Model.php:
```diff
@@ -920,25 +920,27 @@ protected static function buildSelectQuery(
-                $nbRelation++;
-            }
-        }
+		if (is_object(static::$_relations)) {
```
Can you use is_array() here as $_relations is an array ?

iNem0o reviewed (changes requested):
Hello @fernandomema 

Thanks for your fix, i will merge it after a small change.

Best regards

fernandomema committed:
Fix PHP warning

Fix PHP warning:
Warning: Invalid argument supplied for foreach() in %htdocs_path%\vendor\picorm\picorm\src\PicORM\Model.php on line 921

fernandomema reviewed (commented):
Change is_object to is_array method
